README.md

```diff
@@ -2,56 +2,62 @@
 
 This library is currently being used for the course [EE274: Data Compression course, Fall 22](https://stanforddatacompressionclass.github.io/Fall22/) at Stanford University, to augment the lectures and for homeworks: 
-1. The course notes and HWs can be found here: [https://stanforddatacompressionclass.github.io/notes/](https://stanforddatacompressionclass.github.io/notes/)
+3. [EE274 Lecture materials (slides etc)](https://stanforddatacompressionclass.github.io/Fall22/lectures/)
+1. [EE274 Course notes (in progress)](https://stanforddatacompressionclass.github.io/notes/)
 2. The video recordings for lectures can be found as a [Youtube Playlist](https://youtube.com/playlist?list=PLv_7iO_xlL0Jgc35Pqn7XP5VTQ5krLMOl)
-3. All lecture materials from Fall 22 version of EE 274 can be found here: [https://stanforddatacompressionclass.github.io/Fall22/lectures/](https://stanforddatacompressionclass.github.io/Fall22/lectures/)
+
 
 ## Compression algorithms
 Here is a list of algorithms implemented.
+
+### Prefix-free codes
 - [Huffman codes](compressors/huffman_coder.py)
 - [Shannon codes](compressors/shannon_coder.py)
 - [Fano codes](compressors/fano_coder.py)
 - [Shannon Fano Elias](compressors/shannon_fano_elias_coder.py)
 - [Golomb codes](compressors/golomb_coder.py)
 - [Universal integer coder](compressors/universal_integer_coder.py)
+- [Elias Delta code](compressors/elias_delta_uint_coder.py)
+
+### Entropy coders
 - [rANS](compressors/rANS.py)
 - [tANS](compressors/tANS.py)
 - [Typical set coder](compressors/typical_set_coder.py)
-- [zlib (external)](external_compressors/zlib_external.py)
-- [zstd (external)](external_compressors/zstd_external.py)
 - [Arithmetic coder](compressors/arithmetic_coding.py)
 - [Context-based Adaptive Arithmetic coder](compressors/probability_models.py)
 - [Range coder](compressors/range_coder.py)
-- [Elias Delta code](compressors/elias_delta_uint_coder.py)
-- [LZ77](compressors/lz77.py)
 
+### Universal Lossless compressors
+- [zlib (external)](external_compressors/zlib_external.py)
+- [zstd (external)](external_compressors/zstd_external.py)
+- [LZ77](compressors/lz77.py)
 
-NOTE -> the tests in each file should be helpful as a "usage" example of each of the compressors. More details are also available in the lecture [slides](https://stanforddatacompressionclass.github.io/Fall22/lectures/), and the project wiki
 
+NOTE -> the tests in each file should be helpful as a "usage" example of each of the compressors. More details are on the project wiki.
 
 
 ## Getting started
 - Create conda environment and install required packages:
     ```
     conda create --name myenv python=3.8.2
     conda activate myenv
-    python -m pip install -r requirements.txt
+    
     ```
-- Add path to the repo to `PYTHONPATH`:
+- Install the `scl` package
     ```
-    export PYTHONPATH=$PYTHONPATH:<path_to_repo>
+    python install -e . #install the package in a editable mode
```

Comment:
```
(scltestenv) ➜  scl_test git:(kedar/scl_package) python install -e .
/usr/local/bin/python3: can't open file '/Users/shubhamchandak/code/scl_test/install': [Errno 2] No such file or directory
```

Reply:
not sure what to do

Reply:
for me `python setup.py install` worked but not sure that's the right thing to do

Reply:
after python setup.py install, when I opened python interpreter and imported scl it worked well (I tested creating a datablock and computing entropy). But then I went to a different directory and tried import scl but that errored. Is that expected?

Reply:
did you do it from the `stanford_compression_library` dir? seems to work for me.

Reply:
Once the package is added to the python env, we should be able to do that from other directories as well.. I am guessing that scl package hasnt been installed properly as of now.